Example/SnapshotTests/BPKMapAnnotationSnapshotTest.m

```diff
@@ -24,7 +24,7 @@
 
 NSString *const ReuseIdentifier = @"Annotation";
 CGFloat const annotationHeight = 65;
-CGFloat const annotationWidth = 100;
+CGFloat const annotationWidth = 120;
```

Comment:
Nit: I believe our convention for constants is something like `BPKAnnotationWidth`.

Should this be a multiple of a spacing value i.e. `xxl * 3`

Reply:
Even though this is just for a test?

Reply:
Doesen't hurt I suppos